builtin/providers/aws/tagsRedshift_test.go

```diff
@@ -0,0 +1,54 @@
+package aws
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestDiffRedshiftTags(t *testing.T) {
+	cases := []struct {
+		Old, New       map[string]interface{}
+		Create, Remove map[string]string
+	}{
+		{
+			Old: map[string]interface{}{
+				"foo": "bar",
+			},
+			New: map[string]interface{}{
+				"bar": "baz",
+			},
+			Create: map[string]string{
+				"bar": "baz",
+			},
+			Remove: map[string]string{
+				"foo": "bar",
+			},
+		},
+		{
+			Old: map[string]interface{}{
+				"foo": "bar",
+			},
+			New: map[string]interface{}{
+				"foo": "baz",
+			},
+			Create: map[string]string{
+				"foo": "baz",
+			},
+			Remove: map[string]string{
+				"foo": "bar",
+			},
```

Comment:
This test case is *exactly* the same as https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/tagsRDS_test.go#L13

So if this isn't correct, then that isn't correct either :)